### PR TITLE
Timezone for container

### PR DESCRIPTION
**Pull request by git001**
The default timezone is utc. Some customers need to have there local timezone. It would be nice to be able to set the timezone at the creation time

### CODE REVIEW

**Comment by omron93** [comment]
Thanks. It looks good to me. Please update petset template also (https://github.com/sclorg/mongodb-container/blob/master/examples/petset/mongodb-petset-persistent.yaml).

@bparees What about OpenShift POV - it is fine with its policies (to set timezone)?

[test]

**Comment by git001** [comment]
done #223

File: 2.4/examples/replica/mongodb-clustered.json [hunk 1]
```diff
@@ -163,6 +168,10 @@
                   {
                     "name": "MONGODB_KEYFILE_VALUE",
                     "value": "${MONGODB_KEYFILE_VALUE}"
+                  },
+                  {
```
Comment (omron93) [inline 2.4/examples/replica/mongodb-clustered.json:172]: This part is missing in #223. @git001 Please add it (TIMEZONE has no effect without this).

**Comment by bparees** [comment]
I don't know that we have a policy, we've discussed similar changes elsewhere a few times and usually conclude it should just be configured by the user.  Since this is just enabling it to be configured by the user (and providing a default), it seems fine.

**Comment by omron93** [comment]
> I don't know that we have a policy, we've discussed similar changes elsewhere a few times and usually conclude it should just be configured by the user

Thanks. I meant your previous decisions about timezone = "policy" :-)

**Comment by omron93** [comment]
[test]

**Comment by omron93** [comment]
Replication using `ReplSet` is removed. Closing, discuss in #223 instead.

**Pull request closed without merging**